### PR TITLE
Release notes for 3.78.0

### PR DESCRIPTION
**Pull request by slevenick**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,31 @@
-## 3.77.0 (Unreleased)
+## 3.78.0 (Unreleased)
+
+IMPROVEMENTS:
+* composer: added validation for `max_pods_per_node` field. ([#3445](https://github.com/hashicorp/terraform-provider-google-beta/pull/3445))
+* servicenetworking: added support for `user_project_override` and `billing_project ` to `google_service_networking_connection` ([#3455](https://github.com/hashicorp/terraform-provider-google-beta/pull/3455))
+
+BUG FIXES:
+* fixed a crash on `azure_blob_storage_data_source` for `google_storage_transfer_job` ([#3447](https://github.com/hashicorp/terraform-provider-google-beta/pull/3447))
```
Comment (c2thorn) [inline CHANGELOG.md:8]:
```suggestion
* storagetransfer: fixed a crash on `azure_blob_storage_data_source` for `google_storage_transfer_job` ([#3447](https://github.com/hashicorp/terraform-provider-google-beta/pull/3447))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,31 @@
...
+* bigquerydatatransfer: fixed a bug where `destination_dataset_id` was required, it is now optional. ([#3438](https://github.com/hashicorp/terraform-provider-google-beta/pull/3438))
```
Comment (c2thorn) [inline CHANGELOG.md:9]:
```suggestion
* bigquery: fixed a bug in `google_bigquery_data_transfer_config` where `destination_dataset_id` was required, it is now optional . ([#3438](https://github.com/hashicorp/terraform-provider-google-beta/pull/3438))
```

**Review by c2thorn — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (rileykarson) [reply CHANGELOG.md:8]: (same for tpg)

File: CHANGELOG.md (hunk 2 above)
Comment (rileykarson) [reply CHANGELOG.md:9]: I think this one is correct, BQ data transfer is a separate API

Comment (c2thorn) [reply CHANGELOG.md:9]: You're totally right. While confirming what tag we used before, I saw that this note was already a part of the previous changelog entry, so I'm editing my suggestion to remove the line entirely

Comment (rileykarson) [reply CHANGELOG.md:9]: Oh, unusual- I wonder what caused it to show up twice.

**Pull request merged**